packages/design-system-web/src/components/geov-entity-class-icon/geov-entity-class-icon.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import { h } from '@stencil/core';
+import { docsTemlpate } from '../../../.storybook/templates/docsTemplate';
+import { stencilWrapper } from '../../../.storybook/lib/stencilWrapper';
+import componentApi from './docs-component-api.md?raw';
+import overview from './docs-overview.md?raw';
+export default {
+  title: 'Data Components/Entity/EntityClassIcon',
+  tags: ['autodocs'],
+  parameters: {
+    viewMode: 'docs',
+    docs: {
+      page: () => docsTemlpate(overview, componentApi),
+    },
+  },
+};
+export const EntityClassIcon = await stencilWrapper(<geov-entity-class-icon classURI="https://ontome.net/ontology/c21"></geov-entity-class-icon>);
```

Comment:
I suggest to rename EntityClassIcon to EntityClassIconPerson:

```suggestion
export const EntityClassIconPerson = await stencilWrapper(<geov-entity-class-icon classURI="https://ontome.net/ontology/c21"></geov-entity-class-icon>);
```

Reply:
Done